external/kafka-0-10-sql/src/main/scala/org/apache/spark/sql/kafka010/KafkaSource.scala

```diff
@@ -88,7 +88,10 @@ private[kafka010] case class KafkaSource(
 
   private val sc = sqlContext.sparkContext
 
-  private val pollTimeoutMs = sourceOptions.getOrElse("kafkaConsumer.pollTimeoutMs", "512").toLong
+  private val pollTimeoutMs = sourceOptions.getOrElse(
+    "kafkaConsumer.pollTimeoutMs",
+    (sc.conf.getTimeAsSeconds("spark.network.timeout", "120s") * 1000).toString
```

Comment:
My thinking was just to make it really obvious it was the same default as spark.network.timeout usually has, since I didn't find a standalone constant somewhere for that value.  If you think it's clearer as ms, I don't feel strongly either way.